auto_submit/lib/git/git_revert_branch_name.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2023 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Wrapper class to create a revert branch that is comprised of the prefix
+/// revert_ and the commit sha so the branch is easily identifiable.
+class GitRevertBranchName {
```

Comment:
Use a function instead to obtain the `branch`? I don't see why we need a class here, or maybe you will extend this class later one?

Reply:
Yeah I can do that. It was more for clarity. If the graphql was working I would not need this.

Reply:
Looking further I am leaving this as a class but consolidated into a different file.